seatunnel-connectors-v2/connector-file/connector-file-oss/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/oss/sink/OssFileSinkFactory.java

```diff
@@ -57,6 +57,8 @@ public OptionRule optionRule() {
                 .required(OssFileSinkOptions.ACCESS_KEY)
                 .required(OssFileSinkOptions.ACCESS_SECRET)
                 .required(OssFileSinkOptions.ENDPOINT)
+                .required(OssFileSinkOptions.SCHEMA_SAVE_MODE)
+                .required(OssFileSinkOptions.DATA_SAVE_MODE)
```

Comment:
```suggestion
                .optional(OssFileSinkOptions.SCHEMA_SAVE_MODE)
                .optional(OssFileSinkOptions.DATA_SAVE_MODE)
```

Reply:
Thank you for your reply. I will handle it